icu_benchmarks/models/ml_models/xgboost.py

```
@@ -68,7 +72,15 @@ def set_model_args(self, model, *args, **kwargs):
         logging.debug(f"Creating model with: {valid_kwargs}.")
         return model(**valid_kwargs)
 
-    def get_feature_importance(self):
+    def _explain_model(self, reps, labels):
         if not hasattr(self.model, "feature_importances_"):
             raise ValueError("Model has not been fit yet. Call fit_model() before getting feature importances.")
-        return self.model.feature_importances_
+        feature_importances = self.model.feature_importances_
```

Ruff (0.8.2)

Local variable `feature_importances` is assigned to but never used

Remove assignment to unused variable `feature_importances`

(F841)